arctic/store/_pickle_store.py

```diff
@@ -54,12 +56,16 @@ def write(self, arctic_lib, version, symbol, item, previous_version):
         collection = arctic_lib.get_top_level_collection()
         # Try to pickle it. This is best effort
         version['blob'] = _MAGIC_CHUNKED
-        pickled = lz4.compressHC(cPickle.dumps(item, protocol=cPickle.HIGHEST_PROTOCOL))
+        try:
+            pickled = lz4.compressHC(cPickle.dumps(item, protocol=cPickle.HIGHEST_PROTOCOL))
+        except OverflowError:
```

Comment:
I don't disagree with you. @TomTaylorLondon insisted they needed this.

I can re-work the code so it always compresses by chunk. I'm not sure how much overhead compressing once is vs compressing multiple  times though. You also get into the situation where the compressed chunks are significantly smaller, leaving strange sized chunks written to mongo (as opposed to the standard 15MB previously). If these concerns arent major, more than happy to change @richardbounds